### ISSUE TITLE
Service is down

### ISSUE THREAD

**Issue by Raymo111** [bluzi/jsonstore#70]
So the site is working, but my endpoints went down, and any newly generated endpoints also say "ok: false". Has my IP been banned?

**Comment by DamienVesper** [comment]
No, endpoints are also down on my end. Any ETA on when the database will be back up?

**Comment by Raymo111** [comment]
Phew, that's a relief. @bluzi can we get an ETA?

**Comment by youngchief-btw** [comment]
Yep, saw a similar issue on my end where if you load the endpoint in the browser it will take forever, and just end in a Heroku error page, haven't tried in my code though,

**Comment by jajoosam** [comment]
@bluzi please fix! Or if not, let us export data?

**Comment by PotatoCurry** [comment]
Experiencing the same problem described in the issue.

**Comment by Albermonte** [comment]
Still not working, any news?

**Comment by brianmueller** [comment]
@bluzi any updates?

**Comment by DamienVesper** [comment]
@bluzi Anything? It's been more than three weeks since this issue arose and some of us have applications that need data to be running through them.

**Comment by Raymo111** [comment]
@DamienVesper @youngchief-btw @jajoosam @PotatoCurry @Albermonte @brianmueller At this point I'm pretty sure @bluzi is not going to do anything about this project. I guess we should all take this as a lesson to not rely on other people's services, and move on.

**Comment by Albermonte** [comment]
Sad to hear that, I have a lot of data there, not good from my side to depend on this service :/ As soon as it becomes avaible again I'll get all the data and migrate it to a more reliable service and always with a backup, lesson learnt

**Comment by brianmueller** [comment]
What are people planning to migrate to? I’m eyeing Firebase, which definitely has the reliability element. But does anyone know of anything perhaps a bit easier?

**Comment by Albermonte** [comment]
@brianmueller I have my eye on https://github.com/vasanthv/jsonbox, maybe even hosting it myself

**Comment by brianmueller** [comment]
@Albermonte thanks! I'm going to try it. It seems to be maintained well.
Thanks @bluzi , it was fun while it lasted! Hope you don't have the coronavirus 🤞

**Comment by navinor** [comment]
yeah jsonstore was really fun to use, it's a shame it went down...
Good luck @bluzi with your future endeavors!

**Comment by neelr** [comment]
#74 is a great alternative with the similar endpoints! I tried using jsonbox, but it uses a different records based system
